inst/tests/tests.Rraw

```diff
@@ -4105,8 +4105,8 @@ test(1196.2, DT[,.N,by=val]$N, INT(1,1,2))
 test(1197.2, DT[.(x),.N], 2L)
 setNumericRounding(old_rounding)
 
-DT = data.table(id=1:2, val1=6:1, val2=6:1)   # 5380
-test(1199, DT[, sum(.SD), by=id], error="GForce sum can only be applied to columns, not .SD or similar.*looking for.*lapply\\(.SD")
+#DT = data.table(id=1:2, val1=6:1, val2=6:1)   # 5380
+#test(1199, DT[, sum(.SD), by=id], error="GForce sum can only be applied to columns, not .SD or similar.*looking for.*lapply\\(.SD")
```

Comment:
I'm not sure this ever should have been an `error`... Behavior in this PR is to degrade from trying to `GForce` `sum(.SD)` and erroring to just never trying to `GForce` in the first place.

The error is nice in that it provides feedback that the user may be doing it wrong...

Potentially we could add some more checks for this specific case and continue to pass this test?